dimension/extraction.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"strings"
 
-	dataset "github.com/ONSdigital/dp-api-clients-go/dataset"
+	dataset "github.com/ONSdigital/dp-api-clients-go/v2/dataset"
```

Comment:
'dataset' at the start of this line is: Redundant alias
It can and should be removed.